arangod/GeneralServer/GeneralServerFeature.cpp

```diff
@@ -109,12 +109,14 @@ GeneralServerFeature::GeneralServerFeature(
 )
     : ApplicationFeature(server, "GeneralServer"),
       _allowMethodOverride(false),
-      _proxyCheck(true) {
+      _proxyCheck(true),
+      _numIoThreads(8) {
```

Comment:
I wonder whether this is the right default. Maybe something depending on the number of cores available might be better.